_posts/2021-01-21-cockpit-236.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Cockpit 236
+author: pitti
+date: 2021-01-07
+tags: cockpit linux
+slug: cockpit-236
+category: release
+summary: Cockpit with SSH key improvements and motd editing
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 236.
+
+In the last two weeks the Cockpit authors primarily worked on some bug fixes, internal refactoring, dropping some obsolete dependencies like [jquery-flot](https://www.npmjs.com/package/jquery-flot), replacing some custom React components with standard [PatternFly](https://www.patternfly.org/v4/) ones, build system simplifications, and test stabilization. These don't have an user-visible effect, but will make Cockpit easier to develop in the future.
```

Comment:
```suggestion
## Bug fixes & performance improvements

In the last two weeks the Cockpit authors primarily worked on some bug fixes, internal refactoring, dropping some obsolete dependencies like [jquery-flot](https://www.npmjs.com/package/jquery-flot), replacing some custom React components with standard [PatternFly](https://www.patternfly.org/v4/) ones, build system simplifications, and test stabilization. These don't have an user-visible effect, but will make Cockpit easier to develop in the future.
```

It feels like this should have a heading. It _doesn't have to be_ in RPM release notes; it can be a website exclusive (or it _could_ be in the notes too). What should it be called though? General? Maintenance? "Bug fixes & performance improvements"?

I went with the generic Google Play Store app update phrase, kind of as a joke. :wink: At least here, it's true and we explain it.

Reply:
BTW: GitHub's diff is funny. I just added a line with text and a blank line between them. But it's like "everything before and after 'fixes' is different". :laughing:

Reply:
Thanks! Moved to separate header, and dropped the "bug fixes" part, as that's a tautology for cockpit releases :-)